pkgs/top-level/python-packages.nix

```
@@ -11987,6 +11987,8 @@ self: super: with self; {
 
   trimesh = callPackage ../development/python-modules/trimesh { };
 
+  trino-python-client = callPackage ../development/python-modules/trino-python-client { };
```

On PyPI it's called `trino`.
```suggestion
  trino = callPackage ../development/python-modules/trino { };
```